testsuite/features/secondary/srv_cobbler_buildiso.feature

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+@scope_cobbler @scope_iso
+Feature: Cobbler buildiso
+
+  Scenario: Log in as testing user in the cobbler buildiso context
+    Given I am authorized as "testing" with password "testing"
+
+  Scenario: Copy cobbler profiles on the server in the cobbler buildiso context
+    When I copy autoinstall mocked files on server
+
+  Scenario: Create a dummy distro in the cobbler buildiso context
+    Given cobblerd is running
+    Then create distro "testdistro" as user "testing" with password "testing"
+
+  Scenario: Create dummy profiles in the cobbler buildiso context
+    Given cobblerd is running
+    And distro "testdistro" exists
+    Then create profile "orchid" for distro "testdistro" as user "testing" with password "testing"
+    Then create profile "flame" for distro "testdistro" as user "testing" with password "testing"
+    Then create profile "pearl" for distro "testdistro" as user "testing" with password "testing"
+
+  Scenario: Check cobbler created a distro and profiles in the cobbler buildiso context
+    When I follow the left menu "Systems > Autoinstallation > Profiles"
+    Then I should see a "testdistro" text
+    And I should see a "orchid" text
+    And I should see a "flame" text
+    And I should see a "pearl" text
+
+  Scenario: Create dummy system in the Cobbler buildiso context
+    Given cobblerd is running
+    And profile "orchid" exists
+    Then create system "testsystem" for profile "orchid" as user "testing" with password "testing"
+    Then I add the Cobbler parameter "name-servers" with value "9.9.9.9" to item "system" with name "testsystem"
```

Comment:
Same as before, this would be an `And` for me

Reply:
Fixed.